.claude/agents/acceptance-test-creator.md

```
@@ -0,0 +1,179 @@
+| OPT-01 | Edge/Negative | [what we verify] | [scenario] |
+
+### Quick Checklist (2 min)
+- [ ] [most important visual/UX check]
+- [ ] [second most important]
+- [ ] [third if needed]
+```
+
+## Strict Limits
+
+| Section | Max Items | When to include |
+|---------|-----------|-----------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)